packages/autotools-scripts/src/cli.ts

```diff
@@ -17,6 +18,18 @@ program
   ssrTest();
 });
 
+program
+.command('a11y')
+.description('test')
+.option('-p, --path <p>', 'project path')
+.option('-i, --impact <i>', 'Only display issues with impact level higher than <i>. Value between 1 (minor) and 4 (critical)')
```

Comment:
I think it would be easier for the users to specify this option as a string: `--impact minor`. Then we also wouldn't need a to convert it from number to string and back.